lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1180,6 +1180,16 @@ defmodule Kernel.ExpansionTest do
     assert_raise CompileError, ~r"unhandled operator ->", fn ->
       expand(quote do: (foo -> bar))
     end
+
+    assert_raise CompileError,
```

Comment:
I was trying to use the `expand` method, but it was swallowing the warning. I couldn't find a nice way to reuse the functions, so I just added a direct call now on d39bd12